### PR TITLE
system tests: more podman-pod tests

### PR DESCRIPTION
**Pull request by edsantiago**
* rename pod-top.bats to pod.bats
* add test for TCP port communication between pods
* add test for various podman-pod-create options

Signed-off-by: Ed Santiago <santiago@redhat.com>

### CODE REVIEW

File: test/system/200-pod.bats [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+    # By default (podman pod create w/ default --infra) there should be
+    # a /pause container.
+    if [ -z "$no_infra" ]; then
+        is "$output" ".*0 \+1 \+0 \+[0-9. ?s]\+/pause" "there is a /pause container"
+    fi
+
+    # Clean up
+    run_podman pod rm -f $podid
+}
```
Comment (edsantiago) [inline test/system/200-pod.bats:52]: [this is unchanged]

**Comment by mheon** [comment]
LGTM

**Comment by edsantiago** [comment]
Error in integration_test_temporary is:
```
Running: podman [options] stats --all --no-stream --format json
         unexpected end of JSON input
```
(x3). Doesn't look like a flake, but it's also pretty unlikely to be anything related to this PR.

**Comment by mheon** [comment]
That one is a flake. Brent has been chasing it on and off for a while, without luck.

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Pull request merged**